addons/l10n_id_pos/tests/test_qris_pos.py

```diff
@@ -232,6 +232,6 @@ def _patched_make_qris_request(endpoint, params):
                 }
         self.main_pos_config.with_user(self.pos_user).open_ui()
         self.main_pos_config.current_session_id.set_opening_control(0, 'notes')
-        with patch('odoo.addons.l10n_id.models.res_bank._l10n_id_make_qris_request', side_effect=_patched_make_qris_request) as patched:
+        with patch('odoo.addons.l10n_id.models.res_partner_bank._l10n_id_make_qris_request', side_effect=_patched_make_qris_request) as patched:
```

Comment:
none of these l10n_id changes seem necessary (though it's probably cleaner) but once res.bank is removed, a file named res_bank would implicitly mean res.partner.bank